From cc012a2af7f7048b276cc1915561059175c56620 Mon Sep 17 00:00:00 2001
Message-Id: <cc012a2af7f7048b276cc1915561059175c56620.1413837857.git.chang-joon.lee@intel.com>
In-Reply-To: <006497b7323eca32c2f489a1a9f662743c0d16d2.1413837855.git.chang-joon.lee@intel.com>
References: <006497b7323eca32c2f489a1a9f662743c0d16d2.1413837855.git.chang-joon.lee@intel.com>
From: Kristen Carlson Accardi <kristen@linux.intel.com>
Date: Thu, 12 Jun 2014 08:35:44 -0700
Subject: [PATCH 144/312] FROM_UPSTREAM [VPG]: drm/i915: disable power wells
 on suspend

We want to make sure everything is disabled and at its lowest power when
freezing.

FOR: LCK-1074
Change-Id: I2ef288a2a8ab11c282ca51c2248a2a5d2f87c578
Reviewed-by: Imre Deak <imre.deak@intel.com>
Signed-off-by: Kristen Carlson Accardi <kristen@linux.intel.com>
Signed-off-by: Jesse Barnes <jbarnes@virtuousgeek.org>
Signed-off-by: Daniel Vetter <daniel.vetter@ffwll.ch>
---
 drivers/gpu/drm/i915/i915_drv.c |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/gpu/drm/i915/i915_drv.c b/drivers/gpu/drm/i915/i915_drv.c
index cfbc023..c1b30b4 100644
--- a/drivers/gpu/drm/i915/i915_drv.c
+++ b/drivers/gpu/drm/i915/i915_drv.c
@@ -559,6 +559,8 @@ static int i915_drm_freeze(struct drm_device *dev)
 
 	dev_priv->suspend_count++;
 
+	intel_display_set_init_power(dev_priv, false);
+
 	return 0;
 }
 
-- 
1.7.9.5

